server/user.go

```diff
@@ -45,10 +45,13 @@ func GetFeed(c *gin.Context) {
 		user.Synced = time.Now().Unix()
 		store.FromContext(c).UpdateUser(user)
 
+		config := ToConfig(c)
+
 		sync := syncer{
 			remote: remote.FromContext(c),
 			store:  store.FromContext(c),
 			perms:  store.FromContext(c),
+			match: NamespaceFilter(config.Orgs),
```

Comment:
Looks like `OwnersWhitelist` is never used, nor the related configuration variable. Looks like you abandoned that concept and just used `config.Orgs`? If I think about it, that var is probably enough

Reply:
Adjusted! Sorry about that, missed a few things when splitting the features up into separate PRs. Will go over the other PRs too just to be sure I didn't miss anything else.

Reply:
No worries, this is stellar stuff!

Reply:
I feel the same about your fork! It was fantastic to find it. I was previously running drone 0.2 and in need of an upgrade went to the latest version and only learned about the unusual license (given all the code is available) when looking at the autoscaling component.

Found your project via your blog post detailing a similar license confusion. Happy to contribute to such a lightweight, flexible CI solution that doesn't do pricing based on seats and/or repos/builds.